app/controllers/concerns/foreman/controller/registration_commands.rb

```diff
@@ -16,7 +16,19 @@ def registration_args
   end
 
   def insecure
-    registration_params['insecure'] ? '--insecure' : ''
+    if useWget
+      registration_params['insecure'] ? ' --no-check-certificate' : ''
+    else
+      registration_params['insecure'] ? ' --insecure' : ''
+    end
+  end
+
+  def application
+    useWget ? 'wget --no-verbose -O-' : 'curl -sS'
+  end
+
+  def useWget
```

Comment:
snake_case is the convention.
```suggestion
  def use_wget
```

Reply:
Thanks for the hint!